### PR TITLE
Update findXmasSunday function arguments to (1970, 2017)

### LINKED ISSUE #39995: Rosetta Code: Day of the week test is worded incorrectly
**Describe your problem and how to reproduce it:**

The test statement `findXmasSunday(2008, 2121)` should return `[1977, 1983, 1988, 1994, 2005, 2011, 2016]` doesn’t make sense, since 2008 is after 1977.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/coding-interview-prep/rosetta-code/day-of-the-week

**Tell us about your browser and operating system:**
* Browser Name: Firefox
* Browser Version: 81.0.2
* Operating System: Linux Mint

### PR DESCRIPTION
**Pull request by kecleveland**
Update findXmasSunday function arguments from (2008, 2121) to (1970, 2017) to match expected return.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39995

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on opening your first pull request! 😎 Please note that our workflow will require a second approval before this may be merged.

**Review by majestic-owl448 — APPROVED** [review]
LGTM

congratulation for your first contribution!

**Comment by Ieahleen** [comment]
congratulation for your first merged pull request! 🥳

if you want to contribute more, check the issues labelled "help wanted"!

**Pull request merged**
